Sources/LeafKit/LeafError.swift

```diff
@@ -84,6 +86,8 @@ public struct LeafError: Error {
                 return "\(src) - \(key) cyclically referenced in [\(chain.joined(separator: " -> "))]"
             case .lexerError(let e):
                 return "Lexing error - \(e.localizedDescription)"
+            case .missingParameter(let message):
+                return message
```

Comment:
```suggestion
                return "Expected a parameter but found none"
```
As per my comment in `LeafParser.swift`.